Allen/enumerable.rb

```diff
@@ -0,0 +1,12 @@
+module MyEnumerable
```

Comment:
good idea but try doing a for-loop with self[i]. self will be equal to the array so you can do yield self[i] inside the forloop